src/System.Runtime/tests/System/Lazy.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Runtime.Tests
 {
-    public static class LazyInitializerTests
```

Comment:
@mellinoe @stephentoub do one of you guys plan on merging these tests into our internal test system?